centos7/Dockerfile.upgrade.centos7

```diff
@@ -1,37 +1,45 @@
 ARG BASEOS
 ARG BASEVER
 ARG PG_FULL
+ARG PG_MAJOR
 ARG PREFIX
 FROM ${PREFIX}/crunchy-pg-base:${BASEOS}-${PG_FULL}-${BASEVER}
```

Comment:
Does this work when building with Docker? My understanding is that any `ARG` before `FROM` only applies to `FROM`: https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact

If we're relying on buildah being different, that may have changed recently: [containers/buildah#2220](https://github.com/containers/buildah/pull/2220)

Reply:
I'm fine with we swap the from. That said, we're only building with `buildah`